src/bundle/Resources/views/fieldtypes/preview/content_fields.html.twig

```diff
@@ -208,6 +208,18 @@
             </tr>
         </tbody>
     </table>
+
+    {% if parameters['password_expires_at'] %}
+        <p class="mt-2 mb-0">
+            {% if parameters['is_password_expired'] %}
+                {{ 'ezuser.password_already_expired'|trans|desc('Password has expired') }}
+            {% else %}
+                {{ 'ezuser.password_expire_in'|trans({
+                    '%count%': parameters['password_expires_in'].days
```

Comment:
I think we should stick with `password_expire_in`

Reply:
translation key needs to be changed: `password_expires_in` is the correct form. Fixed in  https://github.com/ezsystems/ezplatform-admin-ui/pull/1060/commits/6ef932bb76c280e35cd453553fcfabbf15dae82b